Apps/Sandcastle/gallery/development/3D Tiles Split.html

```diff
@@ -12,7 +12,7 @@
       name="description"
```

Comment:
Ah it looks like there's one more development Sandcastle that needs updating, see `development/Many Clipping Planes`